src/Platform.Handlers/src/Xamarin.Platform.Handlers/Handlers/View/AbstractViewHandler.cs

```diff
@@ -33,7 +33,7 @@ protected AbstractViewHandler(PropertyMapper mapper)
 			_mapper = _defaultMapper;
 		}
 
-		protected abstract TNativeView CreateView();
+		protected abstract TNativeView CreateNativeView();
```

Comment:
Changed name to CreateNativeView to distinguish this from VirtualView

Reply:
I hadn't thought about possible confusion in this case, but this way it's even more explicit. Looks good to me.